<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Categories</class>
 <widget class="QWidget" name="Categories">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Администрирование - услуги</string>
  </property>
  <widget class="QPushButton" name="changeCategory">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QListWidget" name="categoryList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryPrice">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addCategory">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>190</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>60</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоимость услуги</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryName">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>30</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Наиманование услуги</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteCategory">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>270</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выручка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryPercent">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>119</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>120</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
